Refactor support for workflows

privateip opened the pull request:
The initial support for workflows was based on Automation Studio.  The better approach is to basis the available set of workflows on configured automations (specifically triggers) in Operations Manager.  This updates the tools to only consider API triggers as candidates for executing when reasoning through tasks.

AutomateIP reviewed (approved):
Everything is wonderful. Tested get workflows, start workflows and get job status. Good metadata returned from endpoints perspective and now presents a more curated list of things.

privateip committed:
Updates the jobs tools (#40)

The jobs tools provide tools for getting the list of jobs or details
about a specific job.  This update adds additional documentation around
the tool, specifically its return details.  It also filters out fields
that are not important to understanding the job results.
